### ISSUE TITLE
"Technical" keyboard keys behavior in Windows version

### ISSUE THREAD

**Issue by artemp84** [neovim/neovim#22042]
### Describe the bug

There is a different behavior of Scroll Lock, Pause and Num Lock keys in Neovim on Windows.
On Linux, Scroll Lock toggles Scroll Lock indicator on a keyboard, Pause seems to do nothing and Num Lock toggles numpad functionality and respective indicator on a keyboard. Everything as expected.
On Windows, these keys are registered as some other characters or key combinations.
Also, there are differences between nvim running in terminal emulator (cmd, PowerShell) and in nvim-qt.exe

nvim (run from terminal):
ScrLk   - nothing
Pause   - ^Z
NumLock - nothing

nvim-qt:
ScrLk   - &
Pause   - normal mode - nothing, insert mode - acts like backspace
NumLock - %

### Steps to reproduce

On Windows:

1. run nvim-qt.exe
2. switch to INSERT mode
3. press ScrLk key (& character appears)
4. press Num Lock key (% character appears)
5. press Pause key (last character deleted, cursor goes back one character)

### Expected behavior

- ScrLk key - nothing printed in nvim
- Num Lock key - nothing printed in nvim
- Pause key - nothing printed in nvim

### Neovim version (nvim -v)

0.8.2

### Vim (not Nvim) behaves the same?

na

### Operating system/version

Windows 10 19044.1645

### Terminal name/version

cmd, nvim-qt

### $TERM environment variable

na

### Installation

scoop install neovim

**Comment by zeertzjq** [comment]
> Pause - ^Z

What does this mean? ^Z is inserted into buffer?

**Comment by artemp84** [comment]
Exactly. '^Z' is printed in buffer in INSERT mode or briefly appears in command status? (bottom right screen corner) in NORMAL mode.

**Comment by justinmk** [comment]
Try the development version, or at least 0.9 release. Many improvements to Windows TUI support since then.

**Comment by artemp84** [comment]
I tried latest development version, the issue still persists as described.

**Comment by EyeCon** [comment]
Still an issue with nvim v0.10.1.
